.eslintrc.cjs

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+	root: true,
```

Comment:
```suggestion
```

Reply:
I keep the `root: true` option so that ESLint doesn't look for config files that are outside of the project directory. If we don't put it it'll look all the way to `/`.